### PR TITLE
support diag report

### PR DESCRIPTION
**Pull request by zhangzifa**

### CODE REVIEW

File: take_diagnostic_report [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+DIR=$(cd "$(dirname "$0")"; pwd)
+GET_NODE_KILL=$DIR/get_node_kill
+NODE_KILL=`$GET_NODE_KILL`
+
+$NODE_KILL --diag_report $1
```
Comment (hyj1991) [inline take_diagnostic_report:7]: $1 是什么参数？

Comment (zhangzifa) [reply take_diagnostic_report:7]: pid

Comment (zhangzifa) [reply take_diagnostic_report:7]: 跟 heapdump 一样的

**Review by hyj1991 — APPROVED** [review]
+1

**Comment by hyj1991** [comment]
+1

**Pull request merged**